src/utils.ts

```diff
@@ -0,0 +1,10 @@
+import * as fs from 'fs';
+
+export function exists(file: string) {
```

Comment:
Can just use `fs.existsSync` here

Reply:
huh, I thought it was deprecated. Turns out that's just `fs.exists`. Nice!